src/Healthchecks.php

```diff
@@ -321,6 +325,25 @@ protected function validateCheckName(string $checkName)
 		}
 	}
 
+	private function buildTagFilter(array $tags): string
```

Comment:
fuuu :P, we have guzzle, so You can pass `tags` like this
```php
$guzzle->get($url, [
    'query' => [
        'tag' => array_map($tagList, 'trim'),
    ]
]);
```

Reply:
Unfortunately then it wouldn't work.

For example: 
-`?tag[0]=production&tag[1]=devops` won't work.
-`?tag[]=production&tag[]=devops` won't work.
-`?tag=production&tag=devops` works.

Reply:
[RFC1738](https://tools.ietf.org/html/rfc1738) doesn't describe any "arrays" so You may be right, but either way we can do it much simpler:

```php
$keyValuePairs = array_map($tags, function(string $tag) {
    return ['tag', $tag];
});
// $keyValuePairs[] = ['anotherParam', 'itsValue'];

$guzzle->get($url, [
    'query' => implode('&', array_map($keyValuePairs, function(array $kv) {
        return sprintf('%s=%s', $kv[0], urlencode($kv[1]));
    })),
]);
```